docs/api/emoji.rst

```diff
@@ -0,0 +1,34 @@
+.. SPDX-License-Identifier: MIT
+
+.. currentmodule:: disnake
+
+Emoji
+=====
+
+This section documents everything related to guilds' emojis.
```

Comment:
```suggestion
This section documents everything related to Discord emoji.
```

Reply:
Linking to https://discord.com/developers/docs/resources/emoji would be nice too

Reply:
Done in [`d15ddd7`](https://github.com/DisnakeDev/disnake/pull/722/commits/d15ddd70e20c611070a3445379763a80b07a8c13)